CMakeLists.txt

```diff
@@ -98,6 +98,13 @@ else()
   option(ENABLE_LTO "enable link time optimization" ON)
 endif()
 
+if(CMAKE_SYSTEM_NAME MATCHES "OpenBSD|NetBSD|Windows")
+  message(STATUS "skipping libacl on this sytem: ${CMAKE_SYSTEM_NAME}")
+  option(ENABLE_LIBACL "enable libacl" OFF)
+else()
+  option(ENABLE_LIBACL "enabled libacl" ON)
+endif()
```

Comment:
does this still need to be a cmake option? why not let `find_package(LibACL)` decide it?

Reply:
I just followed it because `libintl` and `libiconv` are cmake option without automatic detection.

Reply:
ok. `libintl` and `libiconv` are that way because of history (Windows support). We will remove those options once we're confident that they are always working on Windows.

In the case of ACL, there's no reason to make it an option: either we find the library or we don't.